src/Apps/Partner/Routes/Works/index.tsx

```diff
@@ -100,10 +108,88 @@ export const ArtworksRefetchContainer = createRefetchContainer(
     `,
   },
   graphql`
-    query WorksQuery($partnerId: String!, $input: FilterArtworksInput) {
+    query WorksFilterRefetchQuery(
```

Comment:
I don't like the partner app excessive directory/nesting, b/c these queries exist in a global namespace, but the Relay rule says this has to start with 'Works' (not even partner).